src/main/java/com/sem/btrouble/BTrouble.java

```diff
@@ -23,6 +23,7 @@
 
     private static HashMap<String, Boolean> preferences;
     private AbstractGame game;
+    @SuppressWarnings("unused")
```

Comment:
This is an Eclipse problem. Sometimes when you initialize a dataset in your constructor, it shows up as unused.